packages/editor/src/components/editor-regions/index.js

```diff
@@ -18,10 +18,16 @@ function EditorRegions( {
 	className,
```

Comment:
Why did you choose the "editor" package. The editor package is about the "post editor" (tied to a WP Post object). 
I was thinking this component could be useful in `edit-widgets` too which doesn't depend on the "editor" package.

In fact, I feel we should try to refactor `edit-widgets` to use it. It could be a good proof that this component is generic enough to be shared.

Reply:
Potentially, it could be in "block-editor" (we can try this but make it experimental as a start). I feel this is more something like: `@wordpress/admin-ui-base` or something like that but maybe we should avoid a new package for just one component for now.

Reply:
> Why did you choose the "editor" package. The editor package is about the "post editor" (tied to a WP Post object).

I thought that's what `post-edit` was about :sweat_smile: But yeah, I wasn't totally sure where to put it. `packages/components/` didn't seem right (that's more for UI controls).

> I was thinking this component could be useful in `edit-widgets` too which doesn't depend on the "editor" package.

Ah yeah, good point :+1: 

> In fact, I feel we should try to refactor `edit-widgets` to use it. It could be a good proof that this component is generic enough to be shared.

As part of this PR? :thinking: Or fine to do it in follow-up?

> Potentially, it could be in "block-editor" (we can try this but make it experimental as a start). I feel this is more something like: `@wordpress/admin-ui-base` or something like that but maybe we should avoid a new package for just one component for now.

Yeah, my thoughts exactly. I was originally moving it into a newly created `editor-scaffolding` package, but decided against it. Anyway, I like the idea of putting it into `block-editor` for starters, and maybe to a newly created package once the need arises.